cypress/integration/12-spotlight/spotlight.spec.ts

```diff
@@ -282,21 +282,41 @@ describe("Spotlight", () => {
         cy.openSpotlightDialog().within(() => {
             cy.spotlightFilter(Filter.People);
             cy.spotlightSearch().clear().type("b");
-            cy.spotlightResults().should("have.length", 2);
-            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "true");
-            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "false");
-            cy.spotlightSearch().type("{downArrow}");
-            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "false");
-            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "true");
-            cy.spotlightSearch().type("{downArrow}");
-            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "false");
-            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "false");
-            cy.spotlightSearch().type("{upArrow}");
-            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "false");
-            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "true");
-            cy.spotlightSearch().type("{upArrow}");
-            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "true");
-            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "false");
+            // our debouncing logic only starts the search after a short timeout,
+            // so we wait a few milliseconds.
+            cy.wait(300);
+            cy.get(".mx_Spinner").should("not.exist").then(() => {
```

Comment:
Cypress doesn't really behave like a promise interface, despite exposing a promise interface, but it doesn't hurt anything.

In theory should be able to just stack the calls and it'll figure it out.